desktop/core/src/desktop/settings.py

```diff
@@ -815,3 +815,12 @@ def tracer():
 except ValueError:
   from desktop.monkey_patches import monkey_patch_md5
   monkey_patch_md5(MODULES_TO_PATCH)
+
+################################################################
+# SLACK API Configurations
+################################################################
+
+SLACK_CLIENT_ID = desktop.conf.SLACK.SLACK_CLIENT_ID.get()
+SLACK_CLIENT_SECRET = desktop.conf.SLACK.SLACK_CLIENT_SECRET.get()
+SLACK_VERIFICATION_TOKEN = desktop.conf.SLACK.SLACK_VERIFICATION_TOKEN.get()
+SLACK_BOT_USER_TOKEN = desktop.conf.SLACK.SLACK_BOT_USER_TOKEN.get()
```

Comment:
Why do we need those? AFAIK this is unrelated to Django?

Reply:
These were related to Slack and have been removed from settings.py